app/models/course.rb

```diff
@@ -493,6 +493,24 @@ class Course < ActiveHash::Base
           ]
         }
       ]
+    }, {
+      id: 13,
+      name: "RUST",
+      title: 'Rust',
+      description: 'This is a Rust event. The focus will be on programming in Rust.',
```

Comment:
Suggest title: 'Rust for programmers'
description: 'Some programming knowledge expected, but you don't yet know Rust'